IdentityCore/tests/mac/MSIDMacKeychainTokenCacheTests.m

```diff
@@ -82,7 +82,11 @@ - (void)setUp
     NSError *error;
     MSIDKeychainUtil *keychainUtil = [MSIDKeychainUtil sharedInstance];
     keychainUtil.teamId = @"FakeTeamId";
-    _dataSource = [MSIDMacKeychainTokenCache new];
+    // To ensure that the _dataSource is initialized, we need mark the loginKeychain as non-empty.
+    // MSIDMacKeychainTokenCache::kLoginKeychainEmpty = @"LoginKeychainEmpty"
+    [[NSUserDefaults standardUserDefaults] setBool:NO forKey:@"LoginKeychainEmpty"];
+    _dataSource = [MSAIMSIDMacKeychainTokenCache new];
+    XCTAssertNotNil(_dataSource);
```

Comment:
I don't think we should XCTAssert in "setup". This kind of asserts are supposed to be used inside tests. I think we should use "assert" instead, so if setup incorrect we will crash the test host.

Reply:
We're already usinhg XCTAssert in setup, see L112.
I'm happy to change it, if you like.